sklearn/cluster/hierarchical.py

```diff
@@ -783,9 +796,11 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         By default, no caching is done. If a string is given, it is the
         path to the caching directory.
 
-    n_components : int, optional
```

Comment:
Oh, I thought that theses should remain for one more version, and then be removed. (Hence I modified it)